cog/ticket.py

```
@@ -31,7 +31,10 @@ async def button_callback(self, button, interaction):
             embed.add_field(name="將於幾秒後刪除", value=" ", inline=False)
             await interaction.response.send_message(embed=embed)
             await asyncio.sleep(3)
-            await interaction.channel.delete()
+            try:
+                await interaction.channel.delete()
+            except discord.Forbidden:
+                await interaction.followup.send("我沒有權限刪除頻道！")
```

Ruff (0.15.20)

[warning] String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)